src/JContainers/src/skse/skse.cpp

```diff
@@ -138,11 +138,19 @@ struct real_api : public skse_api
         return ndx != 0xFF ? make_optional (ndx) : nullopt;
     }
 
-    std::optional<std::uint8_t> loaded_light_mod_index (std::string_view const& name) override
+    /// Asuming modIndex is 16-bit, next 16-bit ones in the memory layout looks like reportint 
+    /// the light weight mod index.
+    std::optional<std::uint16_t> loaded_light_mod_index (std::string_view const& name) override
     {
         using namespace std;
-        auto ndx = DataHandler::GetSingleton ()->GetLoadedLightModIndex (string (name).c_str ());
-        return ndx != 0xFF ? make_optional (ndx) : nullopt;
+        auto modinfo = DataHandler::GetSingleton ()->LookupLoadedLightModByName (string (name).c_str ());
+        if (modinfo)
+        {
+            auto ndx = reinterpret_cast<uint16_t const*> (&modinfo->modIndex);
+            auto light_ndx = ndx + 1;
```

Comment:
+1? The first ESL is supposed to be 000. That number at modIndex+2 is correct as-is.